src/dlgNotepad.cpp

```diff
@@ -42,6 +42,22 @@ dlgNotepad::dlgNotepad(Host* pH)
 {
     setupUi(this);
 
+    checkBox_prependText = new QCheckBox("Prepend");
+    toolBar->addWidget(checkBox_prependText);
```

Comment:
Maybe we can do without the extra click, and if text is present in the prepend box, do the action automatically?